newrelic.js

```
@@ -0,0 +1,13 @@
+exports.config = {
+  app_name: process.env.NEW_RELIC_APP_NAME,
```

Biome (1.9.4)

[error] This object property name should be in camelCase.



(lint/style/useNamingConvention)